docs/source/upgrade.md

```diff
@@ -2,6 +2,83 @@
 
 This pages describes Scylla Operator upgrade procedures.
 
+## `v1.0.0` -> `v1.1.0`
+
+During this update we will change readyness probes for Scylla Operator, and initContainer of Scylla Pods, and
+container images.
+
+1. Get name of StatefulSet managing Scylla Operator
+   ```shell
+   kubectl --namespace scylla-operator-system get sts --selector="control-plane=controller-manager"
+
+   NAME                                 READY   AGE
+   scylla-operator-controller-manager   1/1     95m
+   ```
+
+1. Change readyness probe and used container image by applying following patch:
+    ```yaml
+    apiVersion: apps/v1
+    kind: StatefulSet
+    spec:
+      template:
+        spec:
+          containers:
+          - name: manager
+            image: docker.io/scylladb/scylla-operator:1.1.0
+            livenessProbe:
+              httpGet:
+                path: /healthz
+                port: 8080
+                scheme: HTTP
+            readinessProbe:
+              $retainKeys:
+              - httpGet
+              httpGet:
+                path: /readyz
+                port: 8080
+                scheme: HTTP
+    ```
+    To apply above patch save it to file (`operator-patch.yaml` for example) and apply to Operator StatefulSet:
+    ```shell
+    kubectl -n scylla-operator-system patch sts scylla-operator-controller-manager --patch "$(cat operator-patch.yaml)"
+    ```
+1. For each Scylla Cluster execute below steps. To find your clusters you can use:
+    ```shell
+    $ kubectl -n scylla get ScyllaCluster
+
+    NAME             AGE
+    simple-cluster   109m
+    ```
+
+1. Collect all StatefulSets managing Scylla Pods
+    ```shell
+    $ kubectl --namespace scylla get sts --selector="app=scylla,scylla/cluster=simple-cluster"
+
+    NAME                                  READY   AGE
+    simple-cluster-us-east-1-us-east-1a   3/3     107m
+    ```
+    Depending how much racks you have, you may have multiple StatefulSets.
+1. Update each Statefulset by applying following patch:
```

Comment:
These statefulsets are managed by the operator. Why would we ask users to do the work manually instead of our operator upgrading the operands automatically?

Reply:
Current operator is not able to detect that sidecar image is different #187. Maybe we should include it 1.1. WDYT @tzach ?

Reply:
Make sense
Agree with @tnozicka we should try to use standard K8s procedures. 
How much of an effort is #187?

Reply:
Should be straightforward, the only tricky part is upgrading racks sequentially, but it's not the first time we are doing it. I expect to have PR by the end of this week.